sw/device/tests/example_test_from_rom.c

```diff
@@ -16,7 +16,7 @@
 
 static dif_uart_t uart0;
 
-void _boot_start(void) {
+bool rom_test_main(void) {
```

Comment:
You will need to update [this test too](https://github.com/lowRISC/opentitan/blob/48a3d40d2122cf3f3783314028add56b8f39fd30/sw/device/tests/sim_dv/flash_rma_unlocked_test.c#L243).